src/RamTransforms.h

```diff
@@ -476,13 +476,17 @@ class IndexedInequalityTransformer : public RamTransformer {
         return "IndexedInequalityTransformer";
     }
 
-    // converts a box query into a corresponding filter operation
+    // converts a box query into a corresponding partial box query operation
+    // currently this will turn every box query into a filter operation
     bool transformIndexToFilter(RamProgram& program);
 
 protected:
     bool transform(RamTranslationUnit& translationUnit) override {
+	idxAnalysis = translationUnit.getAnalysis<RamIndexAnalysis>();
```

Comment:
Standard single indentation is 4 spaces. Double indentation is thus 8 spaces. No tabs.